src/react/databrowser/buckets/__tests__/BucketCreate.test.tsx

```diff
@@ -165,4 +178,35 @@ describe('BucketCreate', () => {
     ).toBe(true);
     component.unmount();
   });
+  it('should disable cold location from being able to be selected in storage location', async () => {
+    const component = await reduxMountAct(<BucketCreate />, {
+      configuration: {
+        latest: {
+          locations: {
+            'europe25-myroom-cold': {
+              locationType: 'location-dmf-v1',
+              name: 'europe25-myroom-cold',
+              isCold: true,
+              details: {
+                endpoint: 'ws://tape.myroom.europe25.cnes:8181',
+                repoId: ['repoId'],
+                nsId: 'nsId',
+                username: 'username',
+                password: 'password',
+              },
+            },
+          },
+        },
+      },
+    });
+    await act(async () => {
+      const locationSelector = component.find(
+        'select[placeholder="Location Name"]',
+      );
+      locationSelector.getDOMNode().dispatchEvent(new Event('click'));
+    });
```

Comment:
I would preferably opt to test this using `reduxRender` instead of `reduxMountAct ` as it relies on enzyme whereas `reduxRender` rely on testing-library.